### PR TITLE
fix bug gpload error count

### PR DESCRIPTION
**Pull request by dujialun**
gpload error count is incorrect when more than one segment has format
error, for the cmdtime is different, and only errors with the newest
cmdtime is counted.

So we add startTime which will be used for counting all the errors
occured during the same gpload operation.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@dujialun Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=3629)!

[Click here](https://cla.pivotal.io/sync/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=3629) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@dujialun Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/greenplum?repositoryId=greenplum-db/gpdb&pullRequestId=3629)!

**Comment by pf-qiu** [comment]
Please add test case for this bug.

File: gpMgmt/bin/gpload.py [hunk 1]
```diff
@@ -2409,20 +2401,14 @@ def report_errors(self):
...
     def do_insert(self, dest):
         """
         Handle the INSERT case
         """
-        if self.reuse_tables:
-            queryStr = "select cmdtime from gp_read_error_log('%s') group by cmdtime order by cmdtime desc limit 1" % pg.escape_string(self.extTableName)
-            results = self.db.query(queryStr.encode('utf-8')).getresult()
-            if len(results) > 0:
-                self.cmdtime = (results[0])[0]
-
```
Comment (yydzero) [inline gpMgmt/bin/gpload.py]: any reason to remove this code block?

Comment (dujialun) [reply gpMgmt/bin/gpload.py]: The self.cmdtime and self.lastcmdtime were used for distinguishing errors caused by different gpload executions. But if more than one segment have format errors in one gpload execution, there will be different cmdtime in gp_read_error_log caused by different segment, that's what the bug it is. So we discard the self.cmdtime and self.lastcmdtime and add self.startTime to record the start time of the execution and count all the errors which cmdtime is larger than the self.startTime.

Comment (yydzero) [reply gpMgmt/bin/gpload.py]: Got it, thanks Jialun!

**Comment by pf-qiu** [comment]
LGTM

**Pull request merged**